.env

```
@@ -0,0 +1 @@
+UPDATE_SERVICE_BASE_URL='https://api.flites.app'
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)